.prettierignore

```diff
@@ -1,19 +1,11 @@
-.eslintrc
 *.md
-*.js
-*.vue
-*.css
-*.json
-*.yml
-*.html
 *.lock
 *.svg
-public
```

Comment:
@m4n1ok this might need to be updated to be more precise (Eg. `public/build`, `public/assets`, etc?)

Reply:
By default, I decided to ignore the all public folder. At the end, it makes more sense. If user needs to have Prettier proceed a file, they are still able to override the current config.